src/nat_lib.erl

```diff
@@ -24,20 +24,18 @@ soap_request(Url, Function, Msg0, Options) ->
     Action = "\"urn:schemas-upnp-org:service:WANIPConnection:1#"
              ++ Function ++ "\"",
 
-    Headers = [{"Content-Length", integer_to_list(length(Msg))},
+    Headers = [{"Content-Type", "text/xml; charset=\"utf-8\""},
+               {"Content-Length", integer_to_list(length(Msg))},
                {"User-Agent", "Darwin/10.0.0, UPnP/1.0, MiniUPnPc/1.3"},
                {"SOAPAction", Action},
                {"Connection", "close"},
                {"Cache-Control", "no-cache"},
                {"Pragma", "no-cache"}],
 
-
-    Req = {Url, Headers, "text/xml; charset=\"utf-8\"", Msg},
-
-    case httpc:request(post, Req, [], Options) of
```

Comment:
Provided that the caller passes a profile not used by concurrent requests, would avoiding to place the httpc process in the supervisor solve the issue? Requires somehow determining profile to use in httpc call.

Refs:
* [stand_alone](http://erlang.org/documentation/doc-9.1/lib/inets-6.4.2/doc/html/inets.html#start-3).
* [calling httpc with stand_alone pid](http://erlang.org/documentation/doc-9.1/lib/inets-6.4.2/doc/html/httpc.html#request-5).

Snippet:
```
httpc_request(Method, Request, HTTPOptions, Options, Profile) ->
    {ok, Pid} = inets:start(httpc, [{profile, Profile}], stand_alone),
    Response = httpc:request(Method, Request, HTTPOptions, Options, Pid),
    ok = gen_server:stop(Pid, normal, infinity),
    Response.
```

Past usages that may be referred to:
* https://github.com/aeternity/epoch/commit/81c07f46#diff-fd827a9d10583752b1e3ba0d9fa82fb2L377
* https://github.com/aeternity/epoch/commit/786e84b4